cheatsheets/Session_Management_Cheat_Sheet.md

```diff
@@ -163,6 +163,49 @@ Typically, session management capabilities to track users after authentication m
 - Ensure entire cookie should be encrypted if sensitive data is persisted in the cookie
 - Define all cookies being used by the application, their name and why they are needed
 
+# HTML5 Web Storage API
+
+The Web Hypertext Application Technology Working Group (WHATWG) describes the HTML5 Web Storage APIs, localStorage and sessionStorage, as mechanisms for storing name-value pairs client-side.
+Unlike HTTP cookies, the contents of localStorage and sessionStorage are not automatically shared within requests or responses by the browser and are used for storing data client-side.
+
+## The localStorage API
+
+### Scope
+
+Data stored using the localStorage API is accessible by pages which are loaded from the same protocol, port and host.
+This provides similar access to this data as would be achieved by using the "secure" flag on a cookie, meaning that data stored from https could not be retrieved via http.
+
+### Duration
+
+Data stored using the localStorage API is persisted across browsing sessions, meaning that this data is susceptible to access by other system users. 
+
+### Use Case
+
+WHATWG suggests the use of localStorage for data that needs to be accessed across windows or tabs, across multiple sessions, and where large (multi-megabyte) volumes of data may need to be stored for performance reasons.
+
+## The sessionStorage API
+
+### Scope
+
+The sessionStorage API is accessible only to the window or tab that it was stored from, meaning that Tab 1 cannot access data for Tab 2.
+Also, like the localStorage API, data stored using the sessionStorage API is accessible by pages which are loaded from the same protocol, port and host.
+This provides similar access to this data as would be achieved by using the "secure" flag on a cookie, meaning that data stored from https could not be retrieved by http.
+
+### Duration
+
+The sessionStorage API only stores data for the duration of the current browsing session. Once the tab is closed, that data is no longer retrievable.
+
+### Use Case
+
+Despite the name, WHATWG does not suggest that sessionStorage be used to store session identifiers.
```

Comment:
Add link to the part of the WHATWG  spec where they mention the fact mentioned.

Reply:
Was a negative example, namely that they do not mention using session storage to store session identifiers. I can see how this could be leading, and will remove.